pages/docs/mutation.en-US.md

```diff
@@ -100,7 +100,7 @@ mutate('/api/todos', async todos => {
 
 Most probably, you need some data to update the cache. The data is resolved or returned from the promise or async function you passed to `mutate`.
 
-The function will return an updated document to let `mutate` update the corresponding cache value. It could throw an error somehow, every time when you call it.
+The function passed to `mutate` will return an updated document which is used to update the corresponding cache value. If there is an error thown while executing the function, the error will be returned so it can be handled appropriately.
```

Comment:
```suggestion
The function passed to `mutate` will return an updated document which is used to update the corresponding cache value. If there is an error thown while executing the function, the error will be thrown so it can be handled appropriately.
```